validator/client/metrics.go

```diff
@@ -70,6 +70,28 @@ var (
 			"pubkey",
 		},
 	)
+	// ValidatorProposeExitSuccessVec used to count successful voluntary exit proposals.
+	ValidatorProposeExitSuccessVec = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "validator",
+			Name:      "successful_exit_proposals",
```

Comment:
```suggestion
			Name:      "successful_exit_proposals_total",
```

Let's follow the standard: https://prometheus.io/docs/practices/naming/